File: ios/Classes/PDFViewFlutterPlugin.m
#import "PDFViewFlutterPlugin.h"
#import "FlutterPDFView.h"

@implementation FLTPDFViewFlutterPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    FLTPDFViewFactory* pdfViewFactory = [[FLTPDFViewFactory alloc] initWithMessenger:registrar.messenger];
    [registrar registerViewFactory:pdfViewFactory withId:@"plugins.endigo.io/pdfview"];
}
@end

File: packages/flutter_pdfview/ios/flutter_pdfview/Sources/flutter_pdfview/PDFViewFlutterPlugin.swift
import Flutter
import UIKit

/// Entry point registered by Flutter's generated plugin registrant.
///
/// The Objective-C name is pinned to `FLTPDFViewFlutterPlugin` because that is
/// the `pluginClass` declared in `pubspec.yaml`; `GeneratedPluginRegistrant`
/// looks the class up under exactly that name.
@objc(FLTPDFViewFlutterPlugin)
public final class PDFViewFlutterPlugin: NSObject, FlutterPlugin {
    public static func register(with registrar: FlutterPluginRegistrar) {
        let pdfViewFactory = PDFViewFactory(messenger: registrar.messenger())
        registrar.register(pdfViewFactory, withId: "plugins.endigo.io/pdfview")
    }
}
